common/util/file_util.cc

```diff
@@ -69,34 +69,39 @@ absl::string_view Stem(absl::string_view filename) {
 
 // This will always return an error, even if we can't determine anything
 // from errno. Returns "fallback_msg" in that case.
-static absl::Status CreateErrorStatusFromSysError(const char *fallback_msg,
-                                                  int sys_error) {
-  using absl::StatusCode;
+static absl::Status CreateErrorStatusFromSysError(absl::string_view filename,
+                                                  int sys_error,
+                                                  const char *fallback_msg) {
```

Comment:
Perhaps add a comment about `fallback_msg` not needing to be stable because it will be copied?

Reply:
Done, refined comment
```c++
// Create an error message derived from "sys_error" (which contains an errno
// number). The message includes the filename as prefix.
// If "sys_error" can not be resolved, creates an UNKNOWN message.
// The "filename" and "fallback_msg" will be copied, no need for them to
// stay alive after the call.
```